src/huggingface_hub/keras_mixin.py

```diff
@@ -1,24 +1,37 @@
 import collections.abc as collections
 import json
 import os
+import tempfile
 import warnings
 from pathlib import Path
 from shutil import copytree, rmtree
 from typing import Any, Dict, Optional, Union
+from urllib.parse import quote
 
 import yaml
-from huggingface_hub import ModelHubMixin, snapshot_download
+from huggingface_hub import (
+    CommitOperationDelete,
+    ModelHubMixin,
```

Comment:
Is importing `ModelHubMixin` needed?

Reply:
I did not add it myself (it's just an isort refactoring here). But it seems `ModelHubMixin` is use in the `KerasModelHubMixin` class. Is this a deprecated class ?

If you think yes, I can open an issue to deprecate it.